packages/app/src/systems/Home/pages/Home/Home.tsx

```diff
@@ -1,10 +1,41 @@
+import { Flex } from "@fuel-ui/react";
+
+import { useHome } from "../../hooks/useHome";
+import { AssetsTitle } from "../components/AssetsTitle/AssetsTitle";
+import { HomeActions } from "../components/HomeActions/HomeActions";
+
+import { BalanceWidget } from "~/systems/Account/components/BalanceWidget/BalanceWidget";
```

Comment:
Don't forget to create `index.tsx` in specifics folder to don't have big imports like this, but have something like this instead:
```js
import { BalanceWidget } from '~/systems/Account'
```

Also, the same on `line 4-5` on `AssetsTitle` and `HomeActions` imports above. Should be:
```js
import { AssetsTitle, HomeActions } from '../components'
```